docs/guides/external-node/10_attestator_committee.md

```diff
@@ -0,0 +1,41 @@
+# Attester Committee
+
+## Overview
+
+Attester committee is a subset of ZKSync nodes. 
+After each l1 batch execution, participating nodes sign the execution result and send it back to the main node.
```

Comment:
Maybe "L1 batch" or "L1 batch hash" instead of "execution result"?

Reply:
And they don't send the signature just to the main node, it's broadcasted to the entire network, no?

Reply:
yes, mentioned about the network, thanks! As for execution result - I actually wanted to really say execution result.. in the end this is what's important - technically they sign the commitment, which "sets in stone" how the l1 batch was executed. If there are concerns with this - feel free to reopen